docs/review/api/alfa-rules.api.md

```diff
@@ -576,7 +576,7 @@ export class Stability<S extends string = string> extends Tag<"stability"> {
     // (undocumented)
     equals(value: unknown): value is this;
     // (undocumented)
-    static of<S extends string>(stabiltiy: S): Stability<S>;
+    static of<S extends string>(stability: S): Stability<S>;
```

Comment:
It seems this is the only change to the documentation, nothing to the version. I wonder if I should export version similarly as it was done with `scope` and `stability` here: https://github.com/Siteimprove/alfa/blob/main/packages/alfa-rules/src/index.ts#L6-L7

I'm thinking Dory would probably need it